src/EnergyPlus/PluginManager.hh

```diff
@@ -135,27 +147,27 @@ namespace PluginManagement {
         bool bHasUserDefinedComponentModel = false;
         bool bHasUnitarySystemSizing = false;
 #if LINK_WITH_PYTHON
-        PyObjectWrap pModule = nullptr;  // reference to module
-        PyObjectWrap pClassInstance = nullptr; // reference to instantiated class -- *don't decref until the end of the simulation*
+        PyObject *pModule = nullptr;  // reference to module
```

Comment:
No more *Wrap objects necessary, we can now call directly into the Python DLL again.